src/Sylius/Bundle/UiBundle/Resources/views/Grid/Action/links.html.twig

```diff
@@ -1,16 +1,18 @@
-<div class="ui{% if options.icon is defined %} labeled icon{% endif %} floating dropdown {{ options.class|default('') }} link button">
-    {% if options.icon is defined %}<i class="{{ options.icon }} icon"></i>{% endif %}
-    <span class="text">{{ action.label|trans }}</span>
-    <div class="menu">
-        {% if options.header is defined %}
-        <div class="header">
-            {% if options.header.icon is defined %}<i class="{{ options.header.icon }} icon"></i>{% endif %}
-            {{ options.header.label|trans }}
+{% if options.visible|default(true) %}
```

Comment:
I guess that we don't support `visible` property in options, instead of that we have `enabled` field on action object. So IMO you should remove this if statement.